fix: add likeAnchor API to Card element (#1827)
adobe/spectrum-web-components: packages/card/README.md

</sp-card>
```

## Linking

> An `<sp-card>` can be provided with an `href` attribute in order for it to act as one large anchor element. When leveraging the `href` attribute, the `download`, `target` and `rel` attributes come into play to customize the linking behavior of the element. Use them as follows:

Reviewer: ```suggestion
An `<sp-card>` can be provided with an `href` attribute in order for it to act as one large anchor element. When leveraging the `href` attribute, the `download`, `target` and `rel` attributes customize the card's linking behavior. Use them as follows:
```

<!-- prettier-ignore -->
```html
<sp-card
    heading="Card Title"
    subheading="JPG"
    href="https://opensource.adobe.com/spectrum-web-components"
    target="_blank"
>
    <img slot="cover-photo" src="https://picsum.photos/200/300" alt="Demo Image" />

> </sp-card>

Reviewer: Should we provide examples of `download` and `rel` as well, or are they sufficiently self-explanatory?

Author: I think it's OK to skip here, if you're OK with that.
